GDJS/Runtime/layer.js

```diff
@@ -396,21 +407,44 @@ gdjs.Layer.prototype._setEffectsDefaultParameters = function() {
  * time will be slower if time scale is < 1, faster if > 1.
  * @param {number} timeScale The new time scale (must be positive).
  */
-gdjs.Layer.prototype.setTimeScale = function(timeScale) {
+gdjs.Layer.prototype.setTimeScale = function (timeScale) {
   if (timeScale >= 0) this._timeScale = timeScale;
 };
 
 /**
  * Get the time scale for the objects on the layer.
  */
-gdjs.Layer.prototype.getTimeScale = function() {
+gdjs.Layer.prototype.getTimeScale = function () {
   return this._timeScale;
 };
 
 /**
  * Return the time elapsed since the last frame,
  * in milliseconds, for objects on the layer.
  */
-gdjs.Layer.prototype.getElapsedTime = function() {
+gdjs.Layer.prototype.getElapsedTime = function () {
   return this._runtimeScene.getTimeManager().getElapsedTime() * this._timeScale;
 };
+
+/**
+ * Set the position of layer's camera to follow
+ * base layer's camera.
+ */
+gdjs.Layer.prototype.followBaseLayer = function () {
+  this.setCameraX(this._runtimeScene.getLayer('').getCameraX());
+  this.setCameraY(this._runtimeScene.getLayer('').getCameraY());
+};
+
+/**
+ * @return {number[]} the clear color of layer in the range of [0, 1].
```

Comment:
Always include a description to your documented function:

```
One line explanation
@returns {number[]} probably the same explanation
```

Also explicit the format [r,g,b,a], with components in range [0, 1], because it's not a format used often in the rest of the codebase.